src/common/pageTitle/PageTitle.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { FunctionComponent } from 'react';
+
+import styles from './pageTitle.module.scss';
+
+export interface PageTitleProps {
+  title: string;
+}
+
+const PageTitle: FunctionComponent<PageTitleProps> = ({ title }) => {
+  return <h2 className={styles.pageTitle}>{title.toUpperCase()}</h2>;
```

Comment:
Would `Text` component be useful here?
```TSX
<Text as="h2" weight="normalWeight" size="xxl" className={...}>
```

Reply:
That'll work. The only aspect that it does not support is the color of the text, so I'll put that into the module style